fastlane_core/lib/fastlane_core/device_manager.rb

```diff
@@ -194,19 +194,45 @@ def reset_all
         all.each(&:reset)
       end
 
-      def copy_logarchive(device, dest)
+      def copy_logs(device, log_identity, dest)
+        dest = File.expand_path(dest)
+        os_version = ''
+        Open3.popen3("sw_vers -productVersion") do |stdin, stdout, stderr, wait_thr|
```

Comment:
can you please change it to use `FastlaneCore::CommandExecutor` ? to stay in line with other command calls?